server/src/main/java/org/elasticsearch/cluster/routing/allocation/allocator/DesiredBalanceShardsAllocator.java

```diff
@@ -237,6 +237,7 @@ public DesiredBalanceStats getStats() {
             computationsExecuted.count(),
             computationsConverged.count(),
             desiredBalanceComputer.iterations.sum(),
+            computedShardMovements.sum(),
```

Comment:
We should expose average for the above 2 metrics

Reply:
Re above, what naming should be the best?
```
computation_iterations (already exists)
computation_iterations_avg
computed_shard_movements
computed_shard_movements_avg
```

Reply:
I'm not sure the average (over the life of the master) is all that useful. We usually care more about the average over a particular interval, which needs two API outputs taken at either end of the interval. Given that, I think we already expose enough metrics to compute the average: we divide the (difference in the) total by the (difference in the) number of computations executed, right?